routes/auth-routes.js

```diff
@@ -7,21 +7,24 @@
  */
 var authToken = require('../app/auth-token');
 var config = require('../app/config');
+var link = require('../app/public-link');
 var upload = require('../app/notebook-upload');
 var router = require('express').Router();
 var urljoin = require('url-join');
 
-var GET_URL = urljoin(config.get('PUBLIC_LINK'), '/dashboards');
 var UPLOAD_MESSAGE = 'Notebook successfully uploaded';
 
 /* POST /notebooks/* - upload a dashboard notebook */
 router.post('/notebooks(/*)', authToken, upload, function(req, res) {
+    var publicLink = link(req);
```

Comment:
We discussed offline. Leave this as is. Burden falls on the developer / deployer to set values properly. We can't infer everything in every situation.